### ISSUE TITLE
Non-Mastodon fediverse instances are broken

### ISSUE THREAD

**Issue by picsi** [mbajur/mastodon-tags-explorer#8]
See https://mastodon-tags-explorer.hcxp.co/tags/esperanto

Instance #2 is a GNU social instance, not a mastodonian. My suggestion would be to adapt the URL to the other fediverse sites, instead of restricting it to Mastodon-only and getting a partial view of the hashtag usage across the network.

**Comment by mbajur** [comment]
Yeah, that definitely should be fixed, i'm just not sure how to tell what kind of software each instance is using. Is there any universal way of finding that?

Related to #9

**Comment by picsi** [comment]
FYI, it seems to be working now https://mastodon-tags-explorer.hcxp.co/instances/quitter.se
